perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -108,7 +108,10 @@ sub send() {
             if (defined($row->{item_text})) {
                 # this might throw up the odd false positive but only in cases where the
                 # state has changed and there was already update text
-                if ($row->{item_problem_state} && $last_problem_state ne $row->{item_problem_state}) {
+                if ($row->{item_problem_state} && (
+                    ( $last_problem_state eq '' && $row->{item_problem_state} eq 'confirmed' ) ||
```

Comment:
I think you want to `!` this part of the `if` statement - you /don't/ want to add the line if last is blank and state is confirmed.

Reply:
👍